apps/e2e/cypress/e2e/talentsearch/directive-forms.cy.js

```diff
@@ -0,0 +1,837 @@
+    });
+    cy.findByRole("group", {
+      name: /total contract value/i,
+    }).within(() => {
+      cy.findByRole("radio", { name: "$0 to <$10,000" }).click();
+    });
+    cy.findByRole("group", {
+      name: /the estimated total number of resources expected/i,
+    }).within(() => {
+      cy.findByRole("radio", { name: "1 to 5" }).click();
+    });
+    cy.findByRole("group", {
+      name: /expected to start work in/i,
+    }).within(() => {
+      cy.findByRole("radio", { name: "0 to 3 months" }).click();
+    });
+    cy.findByRole("group", {
+      name: /Commodity type/i,
+    }).within(() => {
+      cy.findByRole("radio", {
+        name: "Other",
+      }).click();
+    });
+    cy.findByRole("textbox", {
+      name: /Please specify the commodity/i,
+    }).type("Please specify the commodity");
+    cy.findByRole("group", {
+      name: /Instrument type/i,
+    }).within(() => {
+      cy.findByRole("radio", { name: "Other" }).click();
+    });
+    cy.findByRole("textbox", {
+      name: /Please specify the instrument/i,
+    }).type("Please specify the instrument");
+    cy.findByRole("group", {
+      name: /Method of supply/i,
+    }).within(() => {
+      cy.findByRole("radio", { name: "Other" }).click();
+    });
+    cy.findByRole("textbox", {
+      name: /Please specify the method/i,
+    }).type("Please specify the method");
+    cy.findByRole("group", {
+      name: /Solicitation procedure/i,
+    }).within(() => {
+      cy.findByRole("radio", { name: "Advance contract award notice" }).click();
+    });
+    cy.findByRole("group", {
+      name: /subject to any trade agreements/i,
+    }).within(() => {
+      cy.findByRole("radio", { name: "Yes" }).click();
+    });
+
+    // requirements
+    cy.findByRole("textbox", {
+      name: /tasks that the contractor is expected to perform/i,
+    }).type("tasks that the contractor is expected to perform");
+    cy.findByRole("group", {
+      name: /specific personnel requirements/i,
+    }).within(() => {
+      cy.findByRole("radio", { name: "Yes" }).click();
+    });
+
+    cy.findByRole("button", { name: "Add personnel requirement" }).click();
+    cy.findByRole("group", { name: "Personnel requirement 1" }).within(() => {
+      cy.findByRole("textbox", {
+        name: /Type of personnel/i,
+      }).type("Type of personnel");
+      cy.findByRole("button", { name: "Add a skill" }).click();
+    });
+
+    cy.findByRole("dialog", { name: "Find a skill" }).within(() => {
+      cy.findByRole("combobox", {
+        name: "Skill category",
+      }).then((dropdown) => {
+        cy.wrap(dropdown).select(1); // All categories
+      });
+      cy.findByRole("combobox", {
+        name: "Skill family",
+      }).then((dropdown) => {
+        cy.wrap(dropdown).select(1); // All families
+      });
+      // TODO: yuck, this selection is awful
+      cy.findAllByRole("combobox", { name: /Skill/i }).then((dropdowns) => {
```

Comment:
I couldn't figure out a reliable query for this element.  :cry:  Skill is a substring of Skill category and Skill family and just "Skill" and "Skill *" didn't work.

Reply:
Duh, it needed to be "Skill*" not "Skill *".  :facepalm: